src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -73,10 +78,18 @@ public void onCommand(Member sender, CommandContext context) {
         builder.addField("User ID", userForInfo.getId(), true);
         builder.addField("Mutual Servers", String.valueOf(userForInfo.getMutualGuilds().size()), true);
 
+        if (member != null) {
+            long userId = Long.parseLong(userForInfo.getId());
+            SecurityLevel userSecurityLevel = CascadeBot.INS.getPermissionsManager().getUserSecurityLevel(userId);
+            if (userSecurityLevel != null) {
+                builder.addField("Special role", FormatUtils.formatEnum(userSecurityLevel), true);
```

Comment:
"Special role" still doesn't stick well with me although I'm not sure what else we could use. "Cascade Official Role" maybe?

Reply:
👍